en/docs/setup/changing-the-hostname.md

```diff
@@ -3,13 +3,34 @@
 This section guides you through changing the hostname of WSO2 Identity
 Server.
 
-1.  Change the hostname ( Example: `is.dev.wso2.com`) in
-    ` <IS_HOME>/repository/conf/deployment.toml`.
+1.  Change the hostname 
+    
+    Server hostname for internal API calls, is by default configured as `localhost`. This configuration is utilized 
+    to build the internal absolute url of a service endpoint, that will be consumed at places that will generate 
+    internal API calls. Hence, when configuring the hostname you can follow one from the below two options 
+    according to your requirement.
+
+    - **Option 1**
+
+    If you are configuring the hostnames in production for a proxied server, you need to configure
```

Comment:
...for a proxied server... -> ...for a load-balanced or proxied deployment...

Reply:
Fixed https://github.com/wso2/docs-is/pull/2279/commits/965827548312dc99c576af89e7dea0fafcbdce7d